react/features/stream-effect/JitsiStreamBlurEffect.js

```diff
@@ -0,0 +1,241 @@
+import JitsiStreamEffect from './JitsiStreamEffect';
```

Comment:
Let's rename the directory to stream-effects. This way we can use it for other effects as well.

Note that the index.js file will have to be removed.